### ISSUE TITLE
[6.*] SCSS font paths

### ISSUE THREAD

**Issue by Eldair** [JeffreyWay/laravel-mix#2555]
-   Laravel Mix Version: 6.0.0-beta.10
-   Node Version: 14.13.0
-   OS: Ubuntu 20

### Description:

I have a app.scss file located in /resources/sass/ and fontawesome files located in /resources/fontawesome/.

Setting `$fa-font-path: '../fontawesome/webfonts';` in my scss file works fine when running `production` but fails when running `watch`.

When running `watch` with that line I get this `Can't resolve '../fontawesome/fontawesome/webfonts/fa-brands-400.eot'` (`production` works fine)

When I remove that line and run `production` I get this `Can't resolve '../webfonts/fa-brands-400.eot'` (now `watch` works fine)

It seems like file paths are resolved differently in those commands.

In version 5.* that line was not needed for both watch and prod.

**Comment by Eldair** [comment]
This is the code from scss
```
$fa-font-path: '../fontawesome/webfonts';

@import '../fontawesome/scss/brands';
@import '../fontawesome/scss/solid';
@import '../fontawesome/scss/light';
@import '../fontawesome/scss/fontawesome';
```

If I put `$fa-font-path: '../fontawesome/webfonts';` after import then the reverse happens.

**Comment by Sn0wCrack** [comment]
I'm also currently facing some issues with fontawesome I wasn't facing before.

I'm currently using the following and it appears to pull the fonts out into my public folder in both develop and production modes:
```
@tailwind base;
@tailwind components;
@tailwind utilities;

$fa-font-path: "~@fortawesome/fontawesome-free/webfonts";
@import '~@fortawesome/fontawesome-free/scss/fontawesome';
@import '~@fortawesome/fontawesome-free/scss/regular';
@import '~@fortawesome/fontawesome-free/scss/solid';
```

I've never had to previously do this, unsure if it's a fontawesome or mix v6 change that's caused that to come up.

Weirdly however, despite this seeming to compile fine, the font face definition never seems to actually cause any fonts to load, and therefore no icons appear.

I've verified that the fonts are indeed at the location that's being specified in the font face src rule as well and it appears to check out as I can navigate to that specified URL and it starts downloading the font.

I'm very unsure why no fonts appears to try to load and why the font face definition appears to be ignored

**Comment by bci24** [comment]
the same issue with laravel mix 6.0.0-beta.14 

I copied the folders fonts and images from public to resources/js and now it compiles.

**Comment by Eldair** [comment]
Update: using full, not relative path works
`$fa-font-path: '/resources/fontawesome/webfonts';`

**Comment by thecrypticace** [comment]
Yes, they should act the same. Would you mind setting up a reproduction repo? I started to set one up for this but ran into some issues. A minimal reproduction would be very helpful.

**Comment by Eldair** [comment]
@thecrypticace sure, here it is https://github.com/Eldair/laravel-mix-repro

**Comment by alanondra** [comment]
Setting `$fa-font-path` to anything doesn't have any effect whatsoever, even setting it as global via the `data` key in `sassOptions`. Even filling the variable with gibberish the value it uses is always the same.

**Comment by Eldair** [comment]
In latest version it works fine when omitting `$fa-font-path` for all commands. Non relative path no longer works.

**Comment by MattRogowski** [comment]
I'm still having this issue, can't import the CSS the same way I always have and no examples I try work.
